TreePiece.cpp

```diff
@@ -3872,33 +3887,32 @@ void TreePiece::doAllBuckets(){
 #endif
 
 #ifdef GPU_LOCAL_TREE_WALK
-  ListCompute *listcompute = (ListCompute *) sGravity;
-  DoubleWalkState *state = (DoubleWalkState *)sLocalGravityState;
+  if (!bUseCpu) {
+    ListCompute *listcompute = (ListCompute *) sGravity;
+    listcompute->enableCpu();
```

Comment:
This is non-obvious.  why set the Cpu flag here?  Is it for the next iteration?  If so, add a comment.

Reply:
Extraneous call from when I was debugging. It has been removed.